app/controllers/comments_controller.rb

```diff
@@ -51,4 +52,19 @@ def set_comment
   def comment_params
     params.require(:comment).permit(:content)
   end
+
+  def destroy_comment
+    @destroy_comment ||= ::Comments::Destroy.call(comment: @comment)
+  end
+
+  def create_comment
+    @create_comment ||= ::Comments::Create.call(task: @task, 
+                                                comment_params: comment_params, 
+                                                user: current_user)
+  end
```

Comment:
Ты предложил хороший вариант, а можно и так расписывать, однако по большей части это зависит от того, как настроен рубокоп
```suggestion
  def create_comment
    @create_comment ||= ::Comments::Create.call(
      task: @task, comment_params: comment_params, user: current_user
    )
  end
```